wallet/src/main/java/com/android/identity_credential/wallet/presentation/OpenID4VPPresentationActivity.kt

```diff
@@ -874,17 +874,17 @@ private class TimeChecks : JWTClaimsSetVerifier<SecurityContext> {
         var exp: Instant? = null
         if (expiration != null) {
             exp = Instant.fromEpochMilliseconds(expiration.time)
-            if (exp >= now) {
-                throw BadJWTException("Expired JWT ($exp >= $now)")
+            if (exp <= now) {
```

Comment:
In the unlikely event that exp == now, should we still consider it expired? Should this be < instead of <=?

Reply:
good point - I'd consider it expired so I'll update